rust/agama-lib/share/profile.schema.json

```diff
@@ -164,6 +164,66 @@
                       "mesh",
                       "ap"
                     ]
+                  },
+                  "hidden": {
+                    "title": "Indicates that the wifi network is not broadcasting it's SSID",
```

Comment:
Should it be "wifi", "Wi-Fi", wireless... In this schema we have been using "wireless" but we can change it. Just for consistency.

Reply:
I think `wireless` is fine. I changed the descriptions accordingly.

Reply:
OK, we will plan for that.